internal/logger/logger.go

```diff
@@ -0,0 +1,103 @@
+package logging
+
+import (
+	"fmt"
+	hclog "github.com/hashicorp/go-hclog"
```

Comment:
@yuriolisa I see you removed Zap, what are the advantages of hclog if compared with zap logger? We chose zap for a reason. It's very solid and widespread, has much bigger community and has bench mark tests for performance if we want to export logs in the future to another system. Besides that, that's the basis for Kubernetes logging in general and controller-runtime in specific. Could we accomplish the same with zap?

Reply:
@acmenezes, I think the `hc-logger` it's simpler to implement, however, I agree with you when we advocate in favor to use the same Kubernetes basis logger and it's undoubtedly faster. Therefore, I switched the PR back to Zap package reading from an environment variable, called: `OPCAP_LOG`.